### DISCUSSION TITLE
Exam results

### DISCUSSION THREAD

**Discussion by ilijafilipovski** [sebastian-hofstaetter/teaching#100] [General]
Dear lecture team, 

Can we expect the exam results before or after the deadline for exercise 2(10.06)?

Thank you.
Best regards,
Ilija

**Comment by sophiaalthammer** [comment]
Hi Ilija,

yes you can expect the results by beginning of next week!

Best,
Sophia

**Reply by sophiaalthammer** [reply]
The results are now online and visible :)
